spec/lex.dd

```diff
@@ -397,8 +397,9 @@ r"ab\n" // string is 4 characters,
 ---------------
 `hello`
 `c:\root\foo.exe`
-`ab\n`  // string is 4 characters,
-        // 'a', 'b', '\', 'n'
+`The "lazy" dog`
+`a"b\n`  // string is 5 characters,
+        // 'a', '"', 'b', '\', 'n'
 ---------------
```

Comment:
Yes, I mean in the comment. Didn't see the other examples. I guess we can interpret the `''` here as orthographical quotation marks, not character literals, so it can stay as it is. Thanks!